### ISSUE TITLE
Feature request: Add possibility that all profiles from group can be opened at once

### ISSUE THREAD

**Issue by Bootta11** [Eugeny/terminus#6650]
Use case: for example you are working on different projects where every project had need to have multiple tabs opened(running commands, logs, monitor system etc). To make easier to switch from project to project make feature to define multiple tab groups per project and open it by 1 click on selected group.

Solution 1: Is it possible to add possibility to open all profiles from group by clicking on group name. Like we had already option to select profile and open new tab with it, just add possibility to click on group name and open tabs for all profiles in group. 

Solution 2: You can maybe add another feature to define some kind of open tab groups so instead of opening all profiles from profile group we will open all profiles from open tab group(this is better because you can define to open some profile or group 2 times or you can define some other custom options). I guess first suggestion is lot easier to implement but this is more customizable.

**Comment by JakeyPrime** [comment]
I run a Cluster of 10 nodes in a testing lab for work and this would be a godsend, we have multiple clusters of 10 nodes that all need to be addressed and "broadcasted" to. This would be excellent to have.

**Comment by davixd** [comment]
I would like to have this feature as well.

**Comment by 171121130** [comment]
me too!

**Comment by whysi** [comment]
Same here, there is also the issue https://github.com/Eugeny/tabby/issues/8157 that refers something similar.
A feature like "open all profiles in this group" would be immensely helpful from a sysadmin point of view.

**Comment by Jackysi** [comment]
I agree. I found Tabby recently and I cannot express how much love I have for this app. It's awesome. There are some features I'd love from Xshell, like Open Group, Send commands to group, etc... Bit offtopic apologies, but I'd like to see this feature as well, it would really make sense for sysadmins

**Comment by zz0rk** [comment]
Is this feature implantation still under consideration? Thank you
